src/modules/core/components/ActionsList/ActionsListItem.tsx

```diff
@@ -287,7 +288,7 @@ const ActionsListItem = ({
               <FormattedMessage
                 id={roleMessageDescriptorId || 'action.title'}
                 values={{
-                  actionType,
+                  actionType: isActionTypeAddSafe ? 'AddSafe' : actionType,
```

Comment:
Are there any reasons why this type cannot be determined in `getActionType` like the types of the other actions? This isn't the first action that uses the same events or it's quite similar to other actions, so I'm wondering why can this be done  in the resolvers when others in a similar context can

Reply:
@ArmandoGraterol I think originally I struggled to see where I'd implement the checks, but after looking again I've seen where they should go and have included them in the transformer (for the Colony Actions list) and the resolver (for the Actions Page).